.github/scripts/pr-find-issues.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
+
+PR="$1"
```

Comment:
There should be no need for quotes here.
```suggestion
PR=$1
```

Reply:
Then again it's decent practice to do it, and also works just fine when it's quoted